scripts/elevators.zs

```diff
@@ -0,0 +1,74 @@
+import crafttweaker.liquid.ILiquidStack;
+import crafttweaker.item.IItemStack;
+
+// remove
+recipes.removeByMod("elevatorid");
+
+// Gray Elevator (Base)
+assembler.recipeBuilder()
+    .inputs(<ore:frameGtInvar> * 2)
+    .inputs(<gregtech:turbine_casing:1>)
+    .inputs(<metaitem:electric.motor.lv> * 2)
+    .inputs(<metaitem:emitter.lv>)
+    .outputs(<elevatorid:elevator_gray> * 4)
+    .duration(200)
+    .EUt(15)
+    .buildAndRegister();
```

Comment:
This recipe is currently more expensive than GTNH. I'm going to suggest a modified version of theirs:
```js
[<ore:plateSteel>, <ore:plateSteel>, <ore:plateSteel>],
[<ore:gearSmallSteel>, <ore:frameSteel>, <ore:gearSmallSteel>],
[<ore:plateSteel>, <ore:plateSteel>, <ore:plateSteel>]
```
This would be shaped and produce 2 elevators.